src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -318,6 +341,13 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\StyleCop.MSBuild.4.7.50.0\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\StyleCop.MSBuild.4.7.50.0\build\StyleCop.MSBuild.Targets'))" />
   </Target>
+  <Import Project="..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
```

Comment:
Anything that uses the EasyTables package, yes.

---
In reply to: [56190135](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56190135) [](ancestors = 56190135)